Content.Server/AlertLevel/ChangeAlertLevelComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.ChangeAlertLevel;
+using Robust.Shared.Audio;
+
+namespace Content.Server.ChangeAlertLevel
+{
+    /// <summary>
+    ///     Sets an alert level when activated
+    /// </summary>
+    [RegisterComponent]
+    [Access(typeof(ChangeAlertLevelSystem))]
+    public sealed partial class ChangeAlertLevelComponent : SharedChangeAlertLevelComponent
+    {
+        [DataField("alertLevelOnActivate")] public string AlertLevelOnActivate = default!;
+        [DataField("textField")] public string TextField = default!;
+        [DataField("clickSound")]
+        public SoundSpecifier ClickSound = new SoundPathSpecifier("/Audio/Machines/button.ogg");
```

Comment:
- Don't specify DataField names manually. They default to camelCase of the c# field name.
- `[DataField]` and `public string ..` on separate lines
- The first 2 fields should probably be `required: true`